app/uk/gov/hmrc/exports/movements/models/notifications/NotificationFactory.scala

```
@@ -35,4 +43,13 @@ class NotificationFactory @Inject()(responseParserFactory: ResponseParserFactory
     )
   }
 
+  private def checkResponseCompliance(conversationId: String, xml: NodeSeq): Unit =
+    responseValidator.validate(xml).recover {
```

Based on similar code I changed yesterday, does this not "swallow" the exception?  Is that what we want?

---

Yes. We only want to log the problem. The whole process should continue.